spec/factories/timecard.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :timecard do
+    person
     start_time "2013-12-23 03:31"
```

Comment:
Please do not add associations into the factory. I've found it to cause problems, because you don't control what gets created. What if person creates an assignment in it's factory, and assignments create a requirement, and a requirement creates a task, etc.
Instead, I prefer to create what I need, or exercise the option of stunning it out.

Reply:
@kgf I changed this back and created a person within the test separately.